app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -785,7 +786,7 @@ private Bitmap getPageBitmap(WebView webView) {
     }
 
     private String saveBitmapToStorage(String fileName, Bitmap bitmap) throws IOException {
-        File folder = new File(Environment.getExternalStorageDirectory(), "Zerda");
+        File folder = StorageUtils.getTargetDirForSaveFile(getActivity(), "Zerda");
         if (!folder.exists() && !folder.mkdir()) {
```

Comment:
and there is `FileUtils.ensureDir(dir)` to use

Reply:
Fixed